### ISSUE TITLE
Upgrading from OrangeHRM 4.x to 5.x

### ISSUE THREAD

**Issue by jameswill** [orangehrm/orangehrm#1373]
Hi,

I just wanted to confirm if OrangeHRM 5.x supports upgrade from 4.x ? As I don't find any documentation related to this topic.

If it supports can anyone please share the links to documents ?

**Comment by RajithaKumara** [comment]
Hi @jameswill,
Yes. OrangeHRM upgrader do support.

**Comment by jameswill** [comment]
> Hi @jameswill, Yes. OrangeHRM upgrader do support.

Hi @RajithaKumara ,

Can you please share the upgrade documentation link for step by step guide to upgrade as there is no such documentation I found on the website.

Also I have downloaded the package from Sourceforge and /lib/confs/sysConf.php file contains version as "5.0-beta"
"**$this->version = '5.0-beta';**" (line no. 30)

Can you please confirm if this is a stable release or not ? if it's stable let me know if I have downloaded the correct package from sourceforge ? [https://sourceforge.net/projects/orangehrm/files/stable/5.0/orangehrm-5.0.zip/download](url)

If not, from where I can get the stable package ?

**Comment by jameswill** [comment]
Just wanted to follow up on this.

**Comment by RajithaKumara** [comment]
Hi @jameswill 
Thanks for pointing out the "$this->version = '5.0-beta';". This file `/lib/confs/sysConf.php` is [deprecated](https://github.com/orangehrm/orangehrm/blob/5.1/lib/confs/sysConf.php#L21) and going to remove in next version. I can confirm [the release](https://sourceforge.net/projects/orangehrm/files/stable/5.0/orangehrm-5.0.zip/download) you are pointing is stable 5.0.

Currently, we don't have full documentation for OrangeHRM 5.x version. But it similler as 4.x https://starterhelp.orangehrm.com/hc/en-us/articles/360020359700-OrangeHRM-Starter-Upgrade-Guide

**Comment by RajithaKumara** [comment]
Hi @jameswill ,
We are actively working on our 5.x development. Will try to publish upgrade guide ASAP.
When it comes to the product stability, we have tested all released features. Whatever the pending features, warned in upgrader. If you are really using Dashboard and Buzz, you may have to wait for next releases.

**Comment by KavindaK7** [comment]
Hi @jameswill, 
The OrangeHRM Starter Upgrade manual is available [here](https://starterhelp.orangehrm.com/hc/en-us/articles/6937346912402-OrangeHRM-Starter-Upgrade-Guide).

**Comment by jameswill** [comment]
Is this the same guide for migration from 4.x as well ?

**Comment by jameswill** [comment]
Also the structure for both the packages are totally different Do we need to overwrite the new package or should we delete all the old directories and upload a fresh 5.x package and run the installer to upgrade ?

**Comment by RajithaKumara** [comment]
> Also the structure for both the packages are totally different Do we need to overwrite the new package or should we delete all the old directories and upload a fresh 5.x package and run the installer to upgrade ?

yes. you have to delete old orangehrm directory (make sure to backup it before delete), and extract latest 5.1 ZIP. Open `index. php`, `installer/index. php` from the browser.

**Comment by jameswill** [comment]
Hi,

It seems the upgrade is working fine. However the last step "Launch OrangeHRM" button is not working. When I click on "Launch OrangeHRM" button after successfully completing the installation It redirects to **_/installer_** and gives the following error.

_**The page isn’t redirecting properly**_

Refer the following images.

![1](https://user-images.githubusercontent.com/6368036/196670322-cbc67dd5-e614-4cf7-ba76-be256104259c.png)

![2](https://user-images.githubusercontent.com/6368036/196670331-73117fc1-efc0-4f50-a00b-0bb0075bb4e4.png)

What could be the issue ?

**Comment by RajithaKumara** [comment]
Hi @jameswill ,
can you directly access the login page and see,
`https://orangehrm.domain.com/web/index.php/auth/login`

**Comment by jameswill** [comment]
HI @RajithaKumara ,

Yes. If I explicitely access https://orangehrm.domain.com/web/index.php/auth/login url then this works. But it should work when I click on "Launch OrangeHRM" button.
